src/test/java/io/jenkins/plugins/forensics/git/blame/GitBlamerTest.java

```diff
@@ -327,6 +327,6 @@ private void verifyResult(final FileBlame request, final int line) {
         assertThat(request.getEmail(line)).isEqualTo(EMAIL + line);
         assertThat(request.getName(line)).isEqualTo(NAME + line);
         assertThat(request.getCommit(line)).isNotBlank().isNotEqualTo(EMPTY); // final getter
```

Comment:
Can you check here for the actual value as well (leftover from my mockito problem)? The random long needs to be a constant.

Reply:
Since the `RevCommit` class can't be mocked (`RevCommit#getCommitTime` and `RevCommit#getName` are final), I unfortunately don't see a way to use a constant here.

But I added a method computing the ID for the commit like JGit does, so now the real commit IDs can be checked.

The random long in the `createCommit`-Method is just the id of the git-tree, not this specific commit, but I replaced it with a fixed value as well to ease the computation of the ID.

Reply:
I see.